__tests__/server.test.js

```diff
@@ -12,45 +13,40 @@ beforeAll(async () => {
 });
 
 
+//NUKES THE TEST TO END
 afterAll( async () =>{
   await sequelizeDatabase.drop();
 } );
 
 describe('Server', () => {
+  //test for proof of life path
   test('handles root path', async () => {
     const response = await mockRequest.get('/');
-    
+
     expect(response.status).toBe(200);
     expect(response.text).toEqual('I AM ALIVE!');
   });
-  
-  test('describes food router', async () => {
-    const response = (await request.post('/food')).send({
-      name: 'Test',
-      category: 'fruit',
-    });
```

Comment:
Keep this part because you need something in your database to run the tests. Also, take the parentheses off before the 'await and before the '.send' part